tests/extractor/test_skip_invalid_yaml.py

```diff
@@ -0,0 +1,66 @@
+# TODO: add test case that ensures it raises an exception if a YAML file is not YAML.
+import pytest
```

Comment:
Forgot to remove a self-reminder:
```suggestion
import pytest
```